pkg/wfexec/session.go

```diff
@@ -612,7 +612,8 @@ func (s *Session) exec(ctx context.Context, log *zap.Logger, st *State) (nxt []*
 	st.created = *now()
 
 	defer func() {
-		reason := recover()
+		var reason interface{}
```

Comment:
what does this have to do with what this PR is about?

Reply:
MB, It is not relevant to this PR.